x/evm/types/access_list_tx_test.go

```diff
@@ -0,0 +1,162 @@
+package types
+
+import (
+	"math/big"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/stretchr/testify/suite"
+	"github.com/tharsis/ethermint/tests"
+)
+
+type AccessListTxTestSuite struct {
```

Comment:
nit, I think we could use the same test suite as the other transaction types

Reply:
Absolutely, I'll change it. 

Reason I added this was, that the VS Code Go extension would not let me debug the tests individually when the test suite is defined in another file.